src/toil/batchSystems/singleMachine.py

```diff
@@ -200,7 +201,8 @@ def shutdown(self):
         self.inputQueue = None
         for i in xrange(self.numWorkers):
             inputQueue.put(None)
-
+        #if self.config.useSharedCache:
```

Comment:
oops, yes that needs to go. It's an artifact of when the cleanup was in Resource. It's hence been moved to abstract job store.